ui/js/controllers/bugfiler.js

```diff
@@ -166,6 +168,11 @@ treeherder.controller('BugFilerCtrl', [
                 descriptionStrings += $scope.modalComment;
             }
 
+            var keywords = "";
+            if($scope.isIntermittent) {
+                keywords += "intermittent-failure";
```

Comment:
Ah ok - I'd just thought from the use of `+=` that you were expecting more.

In which case, something like this might be cleaner:
```js
var keywords = $scope.isIntermittent ? "intermittent-failure" : "";
```